lldb/test/API/commands/command/script/TestCommandScript.py

```diff
@@ -161,6 +161,17 @@ def cleanup():
         )
         self.expect("my_command", substrs=["a.out"])
 
+        # Test that without --overwrite we are not allowed to redefine the command.
+        self.expect(
+            "command script add my_command --class welcome.TargetnameCommand",
+            substrs=[
+                "cannot add command: user command exists and force replace not set",
+                "--overwrite",
+                "settings set interpreter.require-overwrite false",
```

Comment:
You could just check for the start of it if you're trying to save being verbose here. There's only one place the error message is generated, so if we have the start we have the end.

Or just check for the whole thing, it's not much more effort. If you want to be sure anyone updating the message later knows that this test exists and maybe needs updating.

Reply:
My original intent was to test that "overwrite" and "settings set..." were mentioned mentioned _somewhere_ in the message, without having to change the test in case someone tweaks the fine print in the future. But I'll update it and just check the whole message, one can argue that if someone changes the fine print it should cause the test to fail.